zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -32,6 +32,10 @@ class EcommerceCrawlStrategy(str, Enum):
     ignored. Use this when some subCategory links are misidentified by
     ML-extraction."""
 
+    none: str = "none"
```

Comment:
I have no strong opinion on whether it is cleaner to have 2 classes vs 1 extra crawl strategy.

Reply:
> I'm thinking about calling it something else which can give more meaning, like "Direct URLs to Product".

Only in the UI, or also in code? (seems rather long for code).

Reply:
> I have no strong opinion on whether it is cleaner to have 2 classes vs 1 extra crawl strategy 

To clarify on this, 2 classes would also mean 2 templates.

> Only in the UI, or also in code? (seems rather long for code).

Yes, in the code, it could just be `direct_product`.